portfolio/src/main/java/com/google/sps/dao/QuoteDao.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.dao;
+
+import com.google.sps.models.Quote;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class QuoteDao implements IQuoteDao{
+    private final List<Quote> quoteList = new ArrayList<>();
+
+    public QuoteDao() {
```

Comment:
optional suggestion: can we also think about having methods to add quotes of our own, i.e. addquote, or even initialize with the list of quotes sent by the user?

Reply:
Yeah, this can be done I will look into it. By user do you mean anyone or just the owner of the page? Alos do we want the quotes to be persistent?

Reply:
I think persistence will eventually come in next step of walkthroughs.
By user I meant the user of the method, i.e. from the code where you call this QuoteDao() constructor, you are passing a list of quotes also (parameterized constructor), to which your QuoteDao gets initialized to.

Reply:
Okay. Thanks for the suggestion I will make that change :)